internal/storage/vmbus/vmbus.go

```diff
@@ -0,0 +1,43 @@
+// +build linux
+
+package vmbus
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+var Filepathglob = filepath.Glob
+
+// WaitForVMBusDevicePath waits for the vmbus device to exist at /sys/bus/vmbus/devices/<vmbusGUIDPattern>...
+func WaitForVMBusDevicePath(ctx context.Context, vmbusGUIDPattern string) (string, error) {
+	vmBusPath := filepath.Join("/sys/bus/vmbus/devices", vmbusGUIDPattern)
+	return WaitForFileMatchingPattern(ctx, vmBusPath)
+}
+
+// WaitForFileMatchingPattern waits for a single file that matches the given path pattern and returns the full path
+// to the resulting file
+func WaitForFileMatchingPattern(ctx context.Context, pattern string) (string, error) {
+	for {
+		files, err := Filepathglob(pattern)
```

Comment:
Can we just write `filepath.Glob` here?

Reply:
Justin's comment above explains why: https://github.com/microsoft/opengcs/pull/369#discussion_r379255089

Reply:
Oh, I see it's exported and we're using it for some sort of test dependency injection.